manifests/mongos/params.pp

```diff
@@ -0,0 +1,64 @@
+# PRIVATE CLASS: do not use directly
+class mongodb::mongos::params inherits mongodb::globals {
+  if $mongodb::globals::use_enterprise_repo {
+    $edition = 'enterprise'
+  } else {
+    $edition = 'org'
+  }
+
+  $manage_package = pick($mongodb::globals::manage_package, $mongodb::globals::manage_package_repo, false)
+
+  $version = $mongodb::globals::version
+
+  $ensure = true
+
+  $package_ensure = pick($version, true)
+  if $manage_package {
+    $package_name = "mongodb-${edition}-mongos"
+  } else {
+    $package_name = 'mongodb-server'
+  }
+
+  $bind_ip     = undef
+  $port        = undef
+  $config_data = undef
+  $configdb    = '127.0.0.1:27019'
+  $restart     = true
+
+  $service_name   = 'mongos'
+  $service_manage = true
+  $service_enable = true
+  $service_ensure = 'running'
+  $service_status = undef
+
+  # Amazon Linux's OS Family is 'Linux', operating system 'Amazon'.
+  case $::osfamily {
```

Comment:
can you please use: 
```suggestion
  case $facts['os']['family'] {
```

Reply:
I'd prefer to keep the facts refactoring till later. It's still used in a few places like `manifests/params.pp`.